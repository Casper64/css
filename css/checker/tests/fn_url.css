
.no_string {
    background-image: url(https://example.com/images/myImg.jpg);
    background-image: url(data:image/png;base64,iRxVBo);
    background-image: url(myFont.woff);
    background-image: url(#IDofSVGpath);
}

.invalid_child {
    background-image: url(10px);
}