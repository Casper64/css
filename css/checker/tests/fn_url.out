css/checker/tests/fn_url.css:10:27: error: invalid value for function "url"
    8 | 
    9 | .invalid_child {
   10 |     background-image: url(10px);
      |                           ~~~~
   11 | }